CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 cmake_minimum_required(VERSION 3.15)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
This does not belong here. If needed for integration with your IDE or some other tool, the option can be configured locally via `cmake -B build -DCMAKE_SOME_OPTION=MyLocalCustomValue`.

Reply:
As for me it's quite handy to have compile commands for navigation regardless of IDE, but ok, will remove.